charts/flyte-binary/README.md

```diff
@@ -11,6 +11,7 @@ Chart for basic single Flyte executable deployment
 | clusterResourceTemplates.annotations | object | `{}` |  |
 | clusterResourceTemplates.externalConfigMap | string | `""` |  |
 | clusterResourceTemplates.inline | object | `{}` |  |
+| clusterResourceTemplates.inlineConfigMap | string | `""` |  |
```

Comment:
Does this name make sense? It mirrors the functionality of `inline` but allows extra configuration to be specified via a configmap.